staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -410,6 +413,17 @@ type JobSpec struct {
 	// This is on by default.
 	// +optional
 	PodReplacementPolicy *PodReplacementPolicy `json:"podReplacementPolicy,omitempty" protobuf:"bytes,14,opt,name=podReplacementPolicy,casttype=podReplacementPolicy"`
+
+	// ManagedBy field indicates the controller that manages a Job. The k8s Job
+	// controller reconciles jobs which don't have this field at all or the field
+	// value is the reserved string `job-controller.k8s.io`, but skips reconciling
```

Comment:
```suggestion
	// value is the reserved string `kubernetes.io/job-controller`, but skips reconciling
```

Regarding https://github.com/kubernetes/kubernetes/pull/123273/files#r1509501578

Reply:
done